src/test/java/seedu/address/testutil/AssessmentBuilder.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.testutil;
+
+import seedu.address.model.assessment.Assessment;
+import seedu.address.model.assessment.AssessmentName;
+import seedu.address.model.assessment.FullMark;
+import seedu.address.model.assessment.Weightage;
+
+/**
+ * A utility class to help with building Assessment objects.
+ */
+public class AssessmentBuilder {
+
+    public static final String DEFAULT_NAME = "Assignment 1";
```

Comment:
This may not be a good default name, could be confused with a real assignment. maybe use something like "Default Assignment" so that user will know to change it.